internal/cmd/list.go

```diff
@@ -145,3 +101,56 @@ func list(cli *CLI) error {
 
 	return writeKubeconfig(user, destinations.Items, grants.Items)
 }
+
+func getUserDestinationGrants() (*api.User, *api.ListResponse[api.Destination], *api.ListResponse[api.Grant], error) {
```

Comment:
Pulled this out so I can call it from the agent without executing `infra list`. No logical changes.